### PR TITLE
VZ-6698 - update HA pipeline to use prod profile

### PR DESCRIPTION
**Pull request by mgianatagh**
- Update the HA test pipeline to be based on a prod profile instead of dev
- Support longer install timeout (sometimes ha install takes longer than 30 minutes)
- Skip stages after a failure

### CODE REVIEW

File: ci/scripts/prepare_jenkins_at_environment.sh [hunk 1]
```diff
@@ -25,6 +25,7 @@ WILDCARD_DNS_DOMAIN=${2:-"x=nip.io"}
 KIND_NODE_COUNT=${KIND_NODE_COUNT:-1}
 TEST_OVERRIDE_CONFIGMAP_FILE="./tests/e2e/config/scripts/pre-install-overrides/test-overrides-configmap.yaml"
 TEST_OVERRIDE_SECRET_FILE="./tests/e2e/config/scripts/pre-install-overrides/test-overrides-secret.yaml"
+INSTALL_TIMEOUT_VALUE=${INSTALL_TIMEOUT:-30m}
```
Comment (stevshap) [inline ci/scripts/prepare_jenkins_at_environment.sh:28]: What is this doing? Why not just use `INSTALL_TIMEOUT` here?

Comment (mgianatagh) [reply ci/scripts/prepare_jenkins_at_environment.sh:28]: INSTALL_TIMEOUT may not be defined, so in that case it defaults to 30m

File: tests/e2e/ha/pods.go [hunk 2]
```diff
@@ -22,12 +23,15 @@ func EventuallyPodsReady(log *zap.SugaredLogger, cs *kubernetes.Clientset) {
 			return false
 		}
 
+		// Assume all pods are ready.  If debug enabled, log status of each pod that is not ready yet
+		returnValue := true
```
Comment (stevshap) [inline tests/e2e/ha/pods.go:27]: How is this different than the code already there?

**Review by stevshap — APPROVED** [review]

File: tests/e2e/ha/pods.go (hunk 2 above)
Comment (mgianatagh) [reply tests/e2e/ha/pods.go:27]: The difference is that when debug is enabled, the test will log all pods not ready or completed on each iteration.  If any get logged then a value of false needs to be returned.  The previous code was returning false on the first pod it found not ready.

File: ci/scripts/prepare_jenkins_at_environment.sh (hunk 1 above)
Comment (stevshap) [reply ci/scripts/prepare_jenkins_at_environment.sh:28]: Gotcha

File: tests/e2e/ha/pods.go (hunk 2 above)
Comment (stevshap) [reply tests/e2e/ha/pods.go:27]: Makes sense

**Pull request merged**